admin_guide/image_policy.adoc

```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
 
 [[image-policy-configuring-the-image-policy-admission-plug-in]]
 == Configuring the ImagePolicy Admission Plug-in
+To configure which images can run on your cluster, configure the ImagePolicy
+Admission plug-in in the `*_master-config.yaml_*` file. You can set one or more
+rules as required. Following are examples of some of the rules you can set:
 
-To enable this feature, configure the plug-in in `*_master-config.yaml_*`:
+* *Reject images with a particular annotation*:
++
+Use this rule to reject all images that have a specific annotation set on them.
+For example, to reject all images that have the annotation
+`images.openshift.io/deny-execution` create the following rule:
++
+[source, yaml]
+----
+- name: execution-denied
+  onResources:
+  - resource: pods
+  - resource: builds
+  reject: true
+  matchImageAnnotations:
+  - key: images.openshift.io/deny-execution <1>
+    value: "true"
+  skipOnResolutionFailure: true
+----
+<1> This annotation may be set by infrastructure that wishes to flag particular images as dangerous.
 
-.Annotated Example File
-====
+* *Enable user to run images from Docker Hub*:
++
+Use this rule to allow users to use images from docker hub:
++
+[source, bash]
+----
+- name: allow-images-from-dockerhub
+  onResources:
+    - resource: pods
+    - resource: builds
+    matchRegistries:
+    - docker.io
+----
 
+Following is an example configuration for setting multiple ImagePolicy
+addmission plugin rules in the `*_master-config.yaml_*` file:
```

Comment:
s/plugin/plug-in

But also, does this need a heading like the other two examples above?

`
*Setting multiple ImagePolicy plug-ins*
`

Reply:
The previous two are example uses of using a single ImagePolicy plugin, this one is an example about how you can define multiple ImagePolicy plugins in `master-config` file. I will keep this separate,.